### PR TITLE
Convert rafter test to go modules

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: tests/rafter/go.mod [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	github.com/kyma-project/rafter v0.0.0-20191209102906-e1e7b47eddf0
+	github.com/onsi/gomega v1.10.1
+	github.com/pkg/errors v0.9.1
+	github.com/vrischmann/envconfig v1.3.0
+	golang.org/x/oauth2 v0.0.0-20191202225959-858c2ad4c8b6 // indirect
+	golang.org/x/time v0.0.0-20191024005414-555d28b269f0 // indirect
+	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
+	google.golang.org/appengine v1.6.5 // indirect
+	k8s.io/api v0.17.11
+	k8s.io/apimachinery v0.17.12-rc.0
```
Comment (aerfio) [inline tests/rafter/go.mod:18]: bump all those k8s.io deps to v0.17.14

**Review by aerfio — CHANGES_REQUESTED** [review]

File: tests/rafter/go.mod (hunk 1 above)
Comment (dbadura) [reply tests/rafter/go.mod:18]: :)

Comment (dbadura) [reply tests/rafter/go.mod:18]: `https://github.com/kubernetes/utils` is not possible to set fixed version, they don't release it.

**Review by aerfio — APPROVED** [review]

**Pull request merged**